linux_numa_cpu.py

```diff
@@ -18,94 +18,106 @@
-	total_stats[node]['cpu_count'] = core2node.values().count(node)
-	for line in cat('/sys/devices/system/node/node{i}/numastat', i=node).split('\n'):
-		(stat, value) = line.split(' ')
-		total_stats[node][stat] = int(value)
+    total_stats[node]['cpu_count'] = core2node.values().count(node)
+    for line in cat(
+        '/sys/devices/system/node/node{i}/numastat',
+            i=node).split('\n'):
+        (stat, value) = line.split(' ')
+        total_stats[node][stat] = int(value)
 
 
 # Send!
 for (node, node_stats) in total_stats.items():
-	for (key, value) in node_stats.items():
-		print('servers.{host}.system.numa.node{node}.{key} {value} {now}'.format(
-			host=hostname,
-			node=node,
-			key=key,
-			value=value,
-			now=now()))
+    for (key, value) in node_stats.items():
+        print('servers.{host}.system.numa.node{node}.{key} {value} {now}'.format(
+            host=hostname, node=node, key=key, value=value, now=now()))
 
 # VCPU -> node assignment
 
+
 def vm_nodes(conn):
-	result = {}
-	for dom in conn.listAllDomains(libvirt.VIR_CONNECT_LIST_DOMAINS_ACTIVE):
-		# dom.vcpus returns:
-		# ([info, info, ...], [affinity, affinity, ...])
-		# Both lists have len() == number of VCPUs
-		#
-		# Info block:
-		#   (<vcpu id>, <?>, <total cpu time?>, <physical core ID>)
-		# Affinity block is a tuple with a bool for each physical core.
-		vcpu_nodes = [core2node[e[3]] for e in dom.vcpus()[0]]
-		name = dom.name()
-		result[name] =  {}
-		for node in nodes:
-			result[name][node] = vcpu_nodes.count(node)
-	return result
+    result = {}
+    for dom in conn.listAllDomains(libvirt.VIR_CONNECT_LIST_DOMAINS_ACTIVE):
+        # dom.vcpus returns:
+        # ([info, info, ...], [affinity, affinity, ...])
+        # Both lists have len() == number of VCPUs
+        #
+        # Info block:
+        #   (<vcpu id>, <?>, <total cpu time?>, <physical core ID>)
+        # Affinity block is a tuple with a bool for each physical core.
+        vcpu_nodes = [core2node[e[3]] for e in dom.vcpus()[0]]
+        name = dom.name()
+        result[name] = {}
+        for node in nodes:
+            result[name][node] = vcpu_nodes.count(node)
+    return result
 
 conn = libvirt and libvirt.openReadOnly(None)
 if conn is not None:
-	result = vm_nodes(conn)
-	for (guest, assignment) in result.items():
-		for (node, num_vcpu) in assignment.items():
-			print('servers.{host}.virtualisation.vserver.{guest}.numa.node{node}.vcpu_count {value} {now}'.format(
-				host=hostname,
-				guest=guest.replace('.', '_'),
-				node=node,
-				value=num_vcpu,
-				now=now()))
-
+    result = vm_nodes(conn)
+    for (guest, assignment) in result.items():
+        for (node, num_vcpu) in assignment.items():
+            print(
+                'servers.{host}.virtualisation.vserver.{guest}.numa.node{node}.vcpu_count {value} {now}'.format(
+                    host=hostname,
+                    guest=guest.replace(
+                        '.',
+                        '_'),
```

Comment:
Guess you touched this afterwards again, but `guest.replace(...)` is not needed to be in several lines.